components/combobox/themes/express.css

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+@container (--system: express) {
```

Comment:
This feels like not a lot of over-writes for system Express. 
When I wrote the Express implementation for SVG we did have to add more.
Maybe the imports have already all of those things but worth checking.

<img width="913" alt="Screen Shot 2023-01-17 at 17 52 17" src="https://user-images.githubusercontent.com/52184321/213054101-95ead616-abad-4815-aed8-2ebafb4e93bf.png">

Reply:
In the updated code, there are 5 overwrites for Express. These are the only ones I see specified on the Figma (_Colors -> Background and Border_). Are there any more that are missing?

Reply:
I'd say you're good to go on these!